pkg/utils/sysfs.go

```diff
@@ -81,3 +81,55 @@ func SetCPUsScalingMaxFreq(cpus []ID, freq int) error {
 
 	return nil
 }
+
```

Comment:
// UncoreFreqAvailable returns true if OS support for uncore frequency control is available, otherwise false.

Reply:
Thanks for noting this! I added a "docstring comment" yesterday but developing on two machines got out-of-sync and force-pushed over that improvement (with other fixes) 😕 Now fixed (again)